interactions.py

```diff
@@ -0,0 +1,22 @@
+from telegram import Update
+from telegram.constants import ParseMode
+from tlgrm.utils import send_reply_message
+
```

Comment:
Can we have a  dictionary and a set of these message ? instead of
raw pasting them inside functions ?
i mean something like (text_content.py):
```python
TEXT_CONTENT = {
    INIT_USER: {
        "fr": """ your stuff """,
        "en": """ your stuff """,
        "gb": """ your stuff """,
    }
}
# and on the function side:
msg = TEXT_CONTENT["INIT_USER"][USER_LANG]
# with USER_LANG set as you want

# let me know what you think
```